internal/model/provenance.go

```diff
@@ -42,7 +42,8 @@ type ProvenanceIR struct {
 	binaryName               string
 	buildCmd                 *[]string
 	builderImageSHA256Digest *string
-	repoURIs                 *[]string
+	repoURI                  *string
```

Comment:
Just an idea:

From `slsav02.GetGitURIAndDigest`, I am wondering whether it makes sense to call this `gitURI` instead of `repoURI` and drop the concept of "repo" totally?

It seems in most places we are assuming Git (and Github?) already.

Reply:
Good point on making the terminology consistent. I think `repoURI` is more suitable. Change all `GitURI`s to `RepoURI`.